website/content/en/docs/best-practices/common-recommendation.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Common recommendations and suggestions"
+linkTitle: "Common suggestions"
+weight: 2
+description: Common recommendations and suggestions to built solutions with Operator SDK
+---
+
+## Overview
+
+Any recommendations or good practices suggested by the Kubernetes community, such as to develop [Operator pattern][operator-pattern] solutions or to use [controller-runtime][controller-runtime], also are good recommendations for those who are looking to build the operator projects with SDK. Also, see [Operator Best Practices][operator-best-practices].
```

Comment:
I don't understand this at all. You link a bunch of different pages related to operators,  but don't really mention what THIS page is going to be about. Maybe something like

"This page covers common recommendations when developing operators. They can be helpful when whether you are using operator-sdk directly, developing an application that follows the [operator pattern][operator-pattern], or directly using [controller-runtime][controller-runtime]. Also, please see [Operator Best Practices](operator-best-practices]."

Reply:
changed as we spoke about tks for help.